src/esbuild_plugins/build_routes/get_route_html.ts

```diff
@@ -1,40 +1,61 @@
 import { normalize } from "https://deno.land/std@0.177.0/path/mod.ts";
-import { join } from "https://deno.land/std@0.182.0/path/mod.ts";
 import { format } from "npm:prettier";
 
-export const get_route_html = async ({ html, css, base_path, site_dir }: {
+const template = (styles: string, hydrate: string, html: string) =>
```

Comment:
does this mean we lose the ability to have your own template file?

Reply:
I think it's fine if we use `svelte:head` special components